vhost-device-console/README.md

```diff
@@ -40,13 +43,20 @@ vhost-device-console --socket-path=<SOCKET_PATH>
  The localhost's port to be used for each guest, this part will be increased with
  0,1,2..socket_count-1.
 
--- option:: -b, --backend=nested|network
+-- option:: -b, --backend=nested|network|uds
 
   The backend type vhost-device-console to be used. The current implementation
-  supports two types of backends: "nested", "network" (described above).
+  supports 3 types of backends: "nested", "network", "uds".
   Note: The nested backend is selected by default and can be used only when
         socket_count equals 1.
 
+.. option:: -v, --uds-path=uds-file-path
```

Comment:
Why the short is `-v` ?
I'm not a fan of short option, but if really want one, I'd say `-u` makes more sense.

Reply:
delete the short option.